doc/include/api_embedc.inc

```diff
@@ -263,42 +194,33 @@ be [const Tcl_Interp*]. In that case the argument in question is
 [comment ---------------------------------------------------------------------]
 [call [cmd ::critcl::cproc] [arg name] [arg arguments] [arg resulttype]]
 
-This variant of [cmd critcl::cproc] assumes that the functionality to
-connect to is implemented by the C function [arg name] which has the
-signature described by the [arg arguments] and [arg resulttype].
+As documnented below, but used when the C function named [arg name] already
+exists.
 
-[para] It creates only the shim performing the conversions required by
-arguments and result.
 
 [comment ---------------------------------------------------------------------]
 [call [cmd ::critcl::cinit] [arg text] [arg externals]]
 
-This command compiles the C code in [arg text] and [arg externals].
-
-[para] Both arguments have access to all definitions created by the
-previously listed commands, regardless of their placement in the
-[file .critcl] file relative to this command.
-See section [sectref {Runtime Behaviour}] for more details.
-
-[para] The C code in [arg text] is put into the body of the
-initialization function of the shared library backing the
-[file .critcl] file, and is executed when this library is loaded into
-the interpreter.
+Appends the C code in [arg text] and [arg externals], but only after all the
+other fragments appended by the previously-listed commands regardless of their
+placement in the [vset critcl_script] relative to this command.  Thus, all
+theri content is visible.  See also [sectref {Runtime Behaviour}].
```

Comment:
`theri` -> `their`

Reply:
Done with commit 29380521d0873ea1dcc5f8cdff6742b20764473f (Branch `documentation` in my repo, not seen by the PR).